js/lib/eventUtil.js

```diff
@@ -10,6 +10,11 @@ module.exports.isForSecondaryAction = (e) =>
   e.button === 1
 
 module.exports.eventElHasAncestorWithClasses = (e, classesToCheck) => {
+  // DO NOT ADD NEW CHECKS USING THIS METHOD
+  // classNames are changed from dev to prod by Aphrodite
```

Comment:
@cezaraugusto mind specifying Aphrodite version (`1.2.3`) and adding the number of the issue (#10029)?